sdk/interpolate/interpolate.go

```diff
@@ -20,7 +27,16 @@ func Do(input string, vars map[string]string) (string, error) {
 	defaults := make(map[string]string, len(vars))
 	empty := make(map[string]string, len(vars))
 
-	for k, v := range vars {
+	// sort key, to replace the longer variables before
+	// see "same prefix" unit test
+	keys := make([]string, len(vars))
+	for k := range vars {
+		keys = append(keys, k)
```

Comment:
don't use append here because you declare array with make

Reply:
@bnjjj ok, from 549 allocs/op to 548 allocs/op without append :)